.github/workflows/osrm-backend.yml

```diff
@@ -95,6 +95,7 @@ jobs:
         ./scripts/format.sh && ./scripts/error_on_dirty.sh
         node ./scripts/validate_changelog.js
         npm run docs && ./scripts/error_on_dirty.sh
+        npm audit --production
```

Comment:
Will alert if we'll start having this problem again. Unfortunately it can be quite unexpected: e.g. if vulnerability will be found in version of `@mapbox/node-pre-gyp` we use and NPM will start reporting it.

Reply:
`--production` means "audit only non-dev dependencies". Our dev dependencies have a lot of problems at the moment and it is quite difficult to fix.